Makefile.defs

```diff
@@ -151,6 +151,10 @@ else ifeq ($(TARGET), sony-prstux)
 	CHOST?=arm-linux-gnueabihf
 	export SONY_PRSTUX=1
 	export USE_LJ_WPACLIENT=1
+else ifeq ($(TARGET), remarkable)
+	CHOST?=arm-kobo-linux-gnueabihf
```

Comment:
Are you using the kobo toolchain from https://github.com/koreader/koxtoolchain?

Reply:
Ah yea I had forgotten about that, I did use the kobo toolchain from koxtoolchain. I started using the toolchain provided by remarkable but it doesn't have a static libstdc++ library (which the koreader build seems to require).

I will take a look at creating a custom toolchain, I think I compared the kobo/remarkable SoCs and decided that there wasn't much difference in the hardware really, but the remarkable is running a much newer version of the Linux kernel so possibly it's worth building a custom toolchain for that reason.

Reply:
[remarkable's cpuinfo](https://remarkablewiki.com/tech/specs) is exactly the same as the [Kobo Forma](https://www.mobileread.com/forums/showpost.php?p=3793588&postcount=14). Not sure about kernel/libc versions in these devices and if is possible to make a new target for both and/or if makes sense. Pinging @NiLuJe

Reply:
Oh, then you're in luck ;).

It's not pluggedd in the koxtoolchain frontend, but the 1.24 branch of my ct-ng tree has a [kobomk7](https://github.com/NiLuJe/crosstool-ng/blob/1.24-kindle/samples/arm-kobomk7-linux-gnueabihf/crosstool.config) sample, which should be *much* closer to that ;).

Reply:
You'll just have to tweak the minimum kernel version (or not, as the Mk7 one is lower and still 4.1.x), and double-check the glibc version, as I'm not sure what poky 2.1.3 was using.

Probably something along those lines, since that's similar the the current Kindle (lab126) TC, IIRC.

Reply:
Probably best to take it as inspiration only, because it's using mainline GCC instead of the final GCC release, and GCC 8/9/10 is still suffering from severe neon performance regressions.